src/js/ctrl/export.js

```diff
@@ -80,6 +81,13 @@ exporter.dataset = function(state, internal, id) {
     spec.transform.push(sort);
   }
 
+  if (transforms) {
+    transforms.forEach(function(element, index, arr) {
+      spec.transform = spec.transform || [];
+      spec.transform.push(element);
+    });
+  }
+
```

Comment:
If you store the transforms under `dataset.transforms` (rather than `._transforms`) you won't need this code anymore -- the transforms will get automatically exported when we call `.toJS()` earlier in this method.